README.md

```diff
@@ -74,6 +74,7 @@ Third-party A-Frame components (of the entity-component system).
 - [aframe-react-boilerplate](https://github.com/ngokevin/aframe-react-boilerplate)
 - [aframe-react-starter](https://github.com/fouad/aframe-react-starter)
 - [React + TweenMax demo](http://codepen.io/cheapsteak/pen/pgbdGa?editors=001) (by [@cheapsteak])
+- [React + Redux demo (incl. switching between 2d and 3d)](https://github.com/RSpace/aframe-meetup-example) (by [@RSpace](https://github.com/RSpace))
```

Comment:
feel free to credit yourself à la https://github.com/aframevr/awesome-aframe/pull/18/files#diff-04c6e90faac2675aa89e2176d2eec7d8R107